packages/react/src/password-input/PasswordInput.tsx

```diff
@@ -66,5 +66,5 @@ export const PasswordInput = forwardRef<HTMLInputElement, PasswordInputProps>(
 
 function usePasswordInputId(idProp: string | undefined) {
 	const autoId = useId(idProp);
-	return idProp ? idProp : `password-input-${autoId}`;
+	return idProp ?? `password-input-${autoId}`;
```

Comment:
[Same issue as here regarding nullish coalescing being the wrong operator.](https://github.com/agriculturegovau/agds-next/pull/1685/files#r1619805454)

Also, the args are setup differently in this edition, not that it matters with the way we're using it.

Reply:
What should our stance be on this? Is there any functional difference to `idProp?` vs `string | undefined`? Using the optional `?` just feels cleaner to me, so should we update all instances we find?

Reply:
>Using the optional ? just feels cleaner to me

Yeah, agreed. It also means we can do `usePasswordInputId()` rather than having to do `usePasswordInputId(undefined)`.